style.css

```diff
@@ -8,12 +8,12 @@ p
 {
   color:#ffffff;
```

Comment:
Isn't this just white? Check all your colors to see if stuff like this is happening.

Reply:
done